packages/node/src/serverless-functions/helpers-web.ts

```diff
@@ -3,42 +3,46 @@ import type { NodeHandler } from '@edge-runtime/node-utils';
 import { buildToNodeHandler } from '@edge-runtime/node-utils';
 import Edge from '@edge-runtime/primitives';
 
-const webHandlerToNodeHandler = buildToNodeHandler(
-  {
-    Headers,
-    ReadableStream,
-    // @ts-expect-error Property 'duplex' is missing in type 'Request'
-    Request: class extends Request {
-      constructor(input: RequestInfo | URL, init?: RequestInit | undefined) {
-        super(input, addDuplexToInit(init));
-      }
+export const createWebExportsHandler = (FetchEvent: typeof Edge.FetchEvent) => {
+  const webHandlerToNodeHandler = buildToNodeHandler(
+    {
+      Headers,
+      ReadableStream,
+      // @ts-expect-error Property 'duplex' is missing in type 'Request'
+      Request: class extends Request {
```

Comment:
I've never seen a class defined inline like this before. Maybe we can do this outside of the function call and then pass it in like normal?

Reply:
it isn't part of the scope of this PR